RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -118,28 +189,31 @@ public bool Generate(ProcessedArgs args)
                 createdJarFilePath = BuildPlugin(definition, args.OutputDirectory);
             }
 
-            LogSummary(createdJarFilePath, generatedSqaleFile, licenseAcceptancePackages);
+            LogSummary(createdJarFilePath, generatedSqaleFile);
 
             return createdJarFilePath != null;
         }
 
-        private void LogSummary(string createdJarFilePath, string generatedSqaleFile, IEnumerable<IPackage> licenseAcceptancePackages)
+        private void LogSummary(string createdJarFilePath, string generatedSqaleFile)
         {
             if (generatedSqaleFile != null)
             {
-                // Log a messsage about the generated sqale file at the end of the process: if we
+                // Log a message about the generated sqale file at the end of the process: if we
```

Comment:
Is this comment still accurate? Multiple plugins are being generated so the message will be output for each plugin, There are a couple of things to think about here: is there any point in generating SQALE files if the user can't use them? Also, the original intent of the comment seems to be have been lost i.e. telling the user about the SQALE file at the end so they don't miss the message.